src/ui/statusbar.c

```diff
@@ -339,21 +356,23 @@ status_bar_draw(void)
             pos = _status_bar_draw_bracket(FALSE, pos, "]");
         }
     }
-
-    wnoutrefresh(statusbar_win);
-    inp_put_back();
+    return pos;
 }
 
+// Checks if there are messages to the side of the current displayed tabs range
```

Comment:
which side?

Reply:
> which side?
outside

Reply:
Its missing in the comment, no? :)

Reply:
> Its missing in the comment, no? :)

but it's kind of self-explanatory with `left_side` argument. I offered before to use doxygen style... I will add that the side is based on the argument.